ImmunityReborn/Assets/Scripts/EnemyShootProjectile.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyShootProjectile : MonoBehaviour
+{
+    public GameObject projectilePrefab;
+    public float speed = 10f;
+    public GameObject player;
+    
+    // Start is called before the first frame update
+    void Start()
+    {
+        InvokeRepeating("ShootAtPlayer", 1f, 2f);
+    }
+
+    private void ShootAtPlayer()
+    {
+        Vector3 playerDirection = player.transform.position - transform.position;
+        GameObject projectile = Instantiate(projectilePrefab, transform.position, transform.rotation);
+        projectile.GetComponent<Rigidbody>().velocity = playerDirection;
+    }
```

Comment:
It's never good to let projectiles live forever, it will just continue to eat more memory. 
Add something simple to make sure the projectile's life ends after some time.

`Destroy( projectile, 5f ); // Destroy the projectile after 5 seconds.`